tests/unit/composables/LetoModelizerApi.test.js

```diff
@@ -99,4 +100,26 @@ describe('User Authentication', () => {
       expect(res.results).toEqual(roles);
     });
   });
+
+  describe('Test function: manageError', () => {
+    it('should throw an error on not 503', async () => {
+      let error = null;
+      try {
+        manageError({ response: { status: 504 } });
+      } catch (e) {
+        error = e;
+      }
+      expect(error).not.toBeNull();
+    });
+
+    it('should no throw an error on 503', async () => {
```

Comment:
should not throw an error when status is 503

Reply:
keep justine sentence

Reply:
should not*

Reply:
should not throw an error if status is 503